### PR TITLE
feat: add heading

### PR DESCRIPTION
**Pull request by anujapandeyy**
# Description:
This pull request adds a feature: the addition of a heading to the HTML file.

### Details:

- Added an  '< h1>'  heading to the HTML file that says "zilla zilla".

- Added an  '< h2>'  heading to the file that says "KZILLA".

### BEFORE:
image

### AFTER:
image

### Checklist
- [ ] My code follows the style guidelines of this project.
- [ ] The code follows HTML best practices and coding standards.
- [ ] I have performed a self review of my own code.

### CODE REVIEW

**Comment by Manaya20** [comment]
gg!!

**Comment by sm-sami** [comment]
You've added a html file with no content?

**Comment by shawshankkumar** [comment]
image


commit names are not upto the mark

File: heading.html
Comment (sm-sami) [inline heading.html:1]: Format this file with prettier

**Review by sm-sami — CHANGES_REQUESTED** [review]

**Review by sm-sami — APPROVED** [review]
LGTM!

**Pull request merged**